src/addons/__tests__/Grid.spec.js

```diff
@@ -231,6 +231,58 @@ describe('Grid', function() {
     });
   });
 
+  describe('Cell Navigation', function() {
+    describe('when cell selection looping is disabled', function() {
+      beforeEach(function() {
+        this.component = this.createComponent({enableCellSelectionLoop: false, enableCellSelect: true});
+      });
+
+      describe('when on last cell', function() {
+        beforeEach(function() {
+          this.component.setState({ selected: { idx: 3, rowIdx: 1 } });
+        });
+        it('selection should move to first cell in next row', function() {
+          this.simulateGridKeyDown('Tab');
+          expect(this.component.state.selected).toEqual({ idx: 0, rowIdx: 2 });
```

Comment:
I think you are confusing the intended functionality this new prop will unlock.
when selection looping is disabled, it would move to next row. When it's enabled, it would move to the first cell in the row. i guess i should give the user 3 rather than 2 navigation options:

1. move to next/previous row
2. move to first/last cell
3. do nothing

Currently it defaults to 1 and 3. does not exist